src/res/data/member.js

```diff
@@ -72,6 +72,14 @@ import SzuTungLin
   from 'src/res/image/member/Szu-Tung-Lin.png'
 import HsiuWenLi
   from 'src/res/image/member/Hsiu-Wen-Li.png'
+import ChihWeiHsu
```

Comment:
Import should be ordered by alphabet.
Fix `ShengHanCian, ChienLiangLiu, MinJiChen, SzuTungLin` as well